Mathlib/Data/Set/Finite.lean

```diff
@@ -1651,6 +1667,14 @@ end Set
 
 namespace Finset
 
+lemma exists_card_eq [Infinite α] : ∀ n : ℕ, ∃ s : Finset α, s.card = n
```

Comment:
Does this come from the fact that `Tendsto Finset.card atTop atTop` in some way? Or, can this make the proof of that easier?

Reply:
Kind of? You would then need to use `Finset.exists_intermediate` to go from an arbitrary size finset back to the size you want. Certainly it doesn't look like an easier proof.